content/applications/hr/recruitment.rst

```diff
@@ -0,0 +1,726 @@
+
+Job section
+^^^^^^^^^^^
+
+The following fields should be pre-populated when creating a new applicant. Altering of these fields
+is allowed if needed.
+
+- :guilabel:`Applied Job`: Select the job position the applicant is applying to from the drop-down
+  menu.
+- :guilabel:`Department`: Select the department the job position falls under from the drop-down
+  menu.
+- :guilabel:`Company`: Select the company the job positions is for using the drop-down menu.
+
+Contract section
+^^^^^^^^^^^^^^^^
+
+- :guilabel:`Expected Salary`: Enter the amount the candidate is requesting for the role in this
+  field. The number should be in a `XX,XXX.XX` format.
+- :guilabel:`Expected Salary - Extra advantages...`: If any extra advantages are requested by the
+  candidate, enter it in this field. This should be short and descriptive, such as `1 week extra
+  vacation` or `dental plan`.
+- :guilabel:`Proposed Salary`: Enter the amount to be offered to the candidate for the role in this
+  field. The number should be in a `XX,XXX.XX` format.
+- :guilabel:`Proposed Salary - Extra advantages...`: If any extra advantages are beng offered to the
+  candidate, enter it in this field. This should be short and descriptive, such as `1 week extra
+  vacation` or `dental plan`.
+- :guilabel:`Availability`: Using the calendar module in the drop-down menu, select the available
+  start date for the candidate.
+
+Application summary tab
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Any additional details or notes that should be added to the applicant's card can be typed into this
+field.
+
+.. image:: recruitment/new-applicant.png
+   :align: center
+   :alt: All the fields for a new applicant form entered.
+
+
+First interview
+---------------
+
+After a candidate has passed the :guilabel:`Initial Qualification` stage, they next move to the
+:guilabel:`First Interview` stage. To move the applicant to the next stage, drag and drop the
+applicant's card to the :guilabel:`First Interview` stage, or click on the :guilabel:`First
+Interview` stage button at the top of the individual applicant's card.
+
+When the candidate's card moves to the :guilabel:`First Interview` stage, an acknowledgement email
+is automatically sent to the candidate. The email is created using an email template titled
+:guilabel:`Applicant: Acknowledgement`. This email template is pre-configured in Odoo, and contains
+dynamic placeholders for the job position, the recruiter's name, and the company's information. Once
+the email is sent, the email appears in the chatter.
+
+.. image:: recruitment/applicant-acknowledgement.png
+   :align: center
+   :alt: An email is automatically sent once an applicant is moved to the First Interview stage. The
+         email is seen in the chatter, and is sent using a template.
+
+Next, an activity is scheduled for the same day, titled :guilabel:`“Send mail for first interview”`.
+This planned activity appears in the chatter. The recruiter then sends an email to the applicant.
```

Comment:
I figured it out! When I was checking in runbot, I moved an app to this stage and i saw a planned activity- BUT it looks like it was set for the initial qualification stage (which is crazy) saying to schedule an interview in 2 weeks. Which I disagree with since you need to determine IF you want to give them an interview. But good catch- I tried several runbots before I figured out what I got wrong!